core/db_models/EEM_Base.model.php

```diff
@@ -773,39 +812,46 @@ public static function reset($timezone = null)
             // because if someone has a reference to it, we can't remove their reference
             // so it's best to keep using the same reference, but change the original object
             // reset all its properties to their original values as defined in the class
-            $r = new ReflectionClass(get_class(static::$_instance));
-            $static_properties = $r->getStaticProperties();
-            foreach ($r->getDefaultProperties() as $property => $value) {
+            $model_class_Name = get_called_class();
```

Comment:
Any reason for not using `static::class` instead of `get_called_class`?

Reply:
cuz I'm old and have a decade of using `get_called_class()` 😆 

`static::class` seems more elegant so I'm going to switch to that, thnx